src/main/java/at/hannibal2/skyhanni/data/MayorAPI.kt

```diff
@@ -0,0 +1,123 @@
+package at.hannibal2.skyhanni.data
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.config.ConfigManager
+import at.hannibal2.skyhanni.data.jsonobjects.local.MayorJson
+import at.hannibal2.skyhanni.events.LorenzTickEvent
+import at.hannibal2.skyhanni.utils.APIUtil
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzUtils.put
+import io.github.moulberry.notenoughupdates.util.SkyBlockTime
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+object MayorAPI {
+    private var lastUpdate = 0L
+    private var dispatcher = Dispatchers.IO
+
+    var rawMayorData: MayorJson? = null
+    var candidates = mapOf<Int, MayorJson.Candidate>()
+    var currentMayor: MayorJson.Candidate? = null
+    var timeTillNextMayor = 0L
+
+    private const val LATE_SPRING = 3
+
+    fun isPerkActive(mayor: String, perk: String) = currentMayor?.let { currentCandidate ->
+        currentCandidate.name == mayor && currentCandidate.perks.any { it.name == perk }
+    } ?: false
+
+    /**
+     * @param input: The name of the mayor
+     * @return: The neu color of the mayor + the name of the mayor; If no mayor was found, it will return "§cUnknown Mayor: §7"
```

Comment:
mistake in the comment (actually my mistake)
```suggestion
     * @return: The neu color of the mayor; If no mayor was found, it will return "§cUnknown Mayor: §7"
```